articles/quickstart/webapp/nextjs/01-login.md

```diff
@@ -23,98 +23,138 @@ useCase: quickstart
+
+export const GET = async function shows() {
+  try {
+    const session = await auth0.getSession();
+
+    if (!session) {
+      return NextResponse.json(
+        { error: 'Not authenticated' },
+        { status: 401 }
+      );
+    }
+
+    const res = new NextResponse();
+    const { token: accessToken } = await auth0.getAccessToken();
+    const apiPort = process.env.API_PORT || 3001;
+    const response = await fetch(`http://localhost:${apiPort}/api/shows`, {
+      headers: {
+        Authorization: `Bearer ${accessToken}`
+      }
+    });
+    const shows = await response.json();
+
+    return NextResponse.json(shows, res);
+  } catch (error) {
+    return NextResponse.json({ error: error.message }, { status: error.status || 500 });
+  }
+};
 ```
 
-The authentication state exposed by `UserProvider` can be accessed in any Client Component using the `useUser()` hook.
+Upon execution the following routes for your customers are available:
 
-:::panel Checkpoint
-Now that you have added the dynamic route and `UserProvider`, run your application to verify that your application is not throwing any errors related to Auth0.
-:::
+- `/auth/login`: The route to perform login with Auth0.
+- `/auth/logout`: The route to log the user out.
+- `/auth/callback`: The route Auth0 will redirect the user to after a successful login.
+- `/auth/profile`: The route to fetch the user profile.
+- `/auth/access-token`: The route to verify the user's session and return an access token (which automatically refreshes if a refresh token is available).
+- `/auth/backchannell-logout`: The route to receive a logout_token when a configured Back-Channel Logout initiator occurs.
+
+To learn more about routing in Auth0, read <a href="https://auth0.com/blog/auth0-stable-support-for-nextjs-app-router/" target="_blank" rel="noreferrer"> Add the dynamic API route</a>.
 
 ## Add Login to Your Application
 
-Users can now log in to your application by visiting the `/api/auth/login` route provided by the SDK. Add a link that points to the login route using an **anchor tag**. Clicking it redirects your users to the Auth0 Universal Login Page, where Auth0 can authenticate them. Upon successful authentication, Auth0 will redirect your users back to your application.
-
-:::note
-Next linting rules might suggest using the `Link` component instead of an anchor tag. The `Link` component is meant to perform <a href="https://nextjs.org/docs/api-reference/next/link" target="_blank" rel="noreferrer">client-side transitions between pages</a>. As the link points to an API route and not to a page, you should keep it as an anchor tag.
-:::
+Users can now log in to your application at `/api/auth/login` route provided by the SDK. Use an **anchor tag** to add a link to the login route, the route redirects your users to the Auth0 Universal Login Page, where Auth0 can authenticate them. Upon successful authentication, Auth0 redirects your users back to your application.
```

Comment:
Run-on sentence: add a link to the login route, the route redirects your 
logout route. The route redirects